python_http_client/client.py

```diff
@@ -117,7 +117,9 @@ def _build_url(self, query_params):
         if query_params:
             url_values = urlencode(sorted(query_params.items()), True)
             url = '{0}?{1}'.format(url, url_values)
-        url = self._build_versioned_url(url) if self._version else self.host + url
+        url = (self._build_versioned_url(url)
+               if self._version
+               else self.host + url)
```

Comment:
It seems like the reason someone would use ternary over a regular `if/else` is that it can fit on one line. In this case, the line exceeds the PEP8 recommendation and now it is back to being multi-line... shouldn't this go back to being a regular `if/else`?

```python
if self._version:
    url = self._build_versioned_url(url)
else:
    url = self.host + url
```

Reply:
You make an excellent point. This is why friends don't let friends edit ternary operators in the middle of the night... Will fix. Thanks!

Reply:
@jlindenger ,

haha no problem!